storage/integration_test.go

```diff
@@ -1334,19 +1337,20 @@ func TestIntegration_Objects(t *testing.T) {
 			if got, want := rc.ContentType(), "text/plain"; got != want {
 				t.Errorf("ContentType (%q) = %q; want %q", obj, got, want)
 			}
-			if got, want := rc.CacheControl(), "public, max-age=60"; got != want {
-				t.Errorf("CacheControl (%q) = %q; want %q", obj, got, want)
-			}
+			// differs
```

Comment:
Thoughts on what's going on here? Should we block off time to pair/discuss?

Reply:
I separated the reader portion out of this test, and tested last modified and cache control separately.